src/ImmersedBoundaries/grid_fitted_bottom.jl

```diff
@@ -126,7 +126,7 @@ end
     # wetting or drying that could happen for a moving grid if we use znode
     z  = rnode(i, j, k, underlying_grid, c, c, c)
     zb = @inbounds ib.bottom_height[i, j, 1]
-    return z ≤ zb
+    return z .≤ zb
```

Comment:
This isn't right because here both `z` and `zb` should be scalars

Reply:
Unless something else is going on...

Reply:
to fix the broadcasting issue I'm making it work with i/j as arrays (the broadcast operator is equivalent on scalars)

Reply:
ah sorry i missed all the other changes here